MetaModel/jamovi/metacorr.u.yaml

```diff
@@ -47,22 +47,20 @@ children:
     stretchFactor: 1
     children:
       - type: LayoutBox
-        margin: none
+        margin: large
         children:
           - type: ComboBox
             name: methodmetacor
-      - type: LayoutBox
-        margin: none
-        children:
+            useSingleCell: true
```

Comment:
the useSingleCell makes the label and the dropdown box stick together rather than align with other boxes around. If you remove the useSingleCell property of make it false, all the dropdowns will align to each other. These are just suggestions.